lib/dndcomment.sty

```diff
@@ -14,19 +14,9 @@
 	fontupper=\fontfamily{lmss}\selectfont,
 	title=#2,
 	parbox = false,
-	colback=commentgreen,
-	colframe=commentgreen,
-	colbacktitle=commentgreen,
+	colback=commentboxcolor,
```

Comment:
Can you change the definition of commentbox to match the definition of paperbox (i.e. with the optional color)?

Reply:
Done.